### PR TITLE
Issue Number #4 added myself as contributor

### PR DESCRIPTION
**Pull request by Vedant-Jayesh-Oza**
Hey @akshitadixit This is the PR for issue number #4 . I have added myself as a contributor in Readme.md. Merge this to the main branch of the repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ A screenshot of the connected system is shown below:
...
+|<a href="https://github.com/Vedant-Jayesh-Oza"><img src="https://avatars.githubusercontent.com/u/75005433?v=4" height="150px" width="150px"></a>| <a href="https://github.com/ShivaSankeerth"><img src="https://avatars.githubusercontent.com/u/29270279?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>|
```
Comment (akshitadixit) [inline README.md:36]:
```suggestion
|<a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>| <a href="https://github.com/ShivaSankeerth"><img src="https://avatars.githubusercontent.com/u/29270279?v=4" height="150px" width="150px"></a>| <a href="https://github.com/Vedant-Jayesh-Oza"><img src="https://avatars.githubusercontent.com/u/75005433?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>| <a href="https://github.com/akshitadixit"><img src="https://avatars.githubusercontent.com/u/56997545?v=4" height="150px" width="150px"></a>|
```

No offence but I am a contributor here too T_T

Comment (akshitadixit) [reply README.md:36]: Please click on add suggestion to batch and  then commit suggestion from the top on this page only.

Comment (Vedant-Jayesh-Oza) [reply README.md:36]: Hey actually I didn't get what you said 😅

Comment (Vedant-Jayesh-Oza) [reply README.md:36]: > Please click on add suggestion to batch and  then commit suggestion from the top on this page only.

Oh okay I removed your name and image and I added mine thats what the problem is ? 😅😅 sorry I didn't knew about this like I thought we need to remove and add ours so sorry 😅😅

Comment (akshitadixit) [reply README.md:36]: No issues. Pull the fresh changes from the report and then add yourself.

Comment (Vedant-Jayesh-Oza) [reply README.md:36]: So I need to make one more branch and then pull upstream and then make changes in the code again but adding myself after your name I need to commit changes and make PR right ?

Comment (Vedant-Jayesh-Oza) [reply README.md:36]: Hey I have dm'd you on slack about the issue I am facing please have a look 😅

Comment (akshitadixit) [reply README.md:36]: Follow these steps:
- `git pull https://github.com/akshitadixit/VCallZer` to pull all new changes from the repo to your local clone.
- Make your changes, if there are any conflicts, resolve them.
- Commit and push

All these changes will be reflected in the same PR. Reach out for any help.

Comment (Vedant-Jayesh-Oza) [reply README.md:36]: I did same but I am facing issues 😅😅

**Pull request merged**